### PR TITLE
workflows/run_tests.sh: --oversubscribe the mpi tests

### PR DESCRIPTION
**Pull request by mhasself**

### CODE REVIEW

**Comment by mhasself** [comment]
I will merge this soon unless @tskisner provides negative comment.

**Review by tskisner — APPROVED** [review]
Sure, this does no harm.  We can merge now and then revisit tests (and release automation) after the toast3 merge.

**Comment by tskisner** [comment]
I'm actually seeing some strange behavior in the other PR unit tests (using the so3g wheels).  Normally we let the author of the PR merge the PR after approval, but since this is one line and might help that other situation, I'm going to merge this now.

**Pull request merged**
